### ISSUE TITLE
Binding properties: Color contrast ratio of sort filter icons is insufficient (2:1)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4662]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
- Test URL: https://clarity.design/documentation/datagrid/binding-properties
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. open the test URL.
2. Check the color contrast ratio of the sort icons.

**Expected result:**
Color contrast icons should be at least 3:1 for non textual informative elements.

**Actual result:**
Color contrast ratio is 2:1.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
